flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -651,14 +651,38 @@
 
 	// ---------------------------- Metrics -----------------------------------
 
+	// Per reporter:
```

Comment:
i think this comment and the "For all reporters" don't stand out enough to be included. My first impression was that these were left-overs.